py_irt/models/one_param_logistic.py

```diff
@@ -211,6 +213,16 @@ def fit_MCMC(self, models, items, responses, num_epochs):
         print(theta_sum)
         print(b_sum)
 
+    def predict(self, models, items, params_from_file=None):
+        "predict p(correct | params) for a specified list of model, item pairs"
```

Comment:
For doc comments, this should be triple quote

Reply:
Updated throughout.